src/main/java/io/knowledgeassets/myskills/server/communityuser/command/CommunityUserCommandService.java

```diff
@@ -109,26 +117,55 @@ public CommunityUser update(String communityId, String userId, CommunityRole rol
 	}
 
 	/**
-	 * Authenticated user leaves the community.
-	 * @param communityId - id of the community to leave
-	 * @param userId - user id
+	 * Kicks out the user from the community.
+	 * @param communityId community ID
+	 * @param userId user ID
 	 */
 	@Transactional
-	public void remove(String communityId, String userId) {
+	public void kickoutUser(String communityId, String userId) {
```

Comment:
I think we don't need two separate methods in this case. The parameterization of the `remove` method controls...

* ... if the executing user is really allowed to perform this operation.
* ... what the type of the generated notification is.

This thought is closely related to my other comment concerned with the consequent shift of business logic to the service beans.

Reply:
With the authorization on the controller layer in mind I understand that there need to be two methods to distinguish the two use cases. Therefore, we can leave it like this. 👍

Reply:
Thank you for the note. We could stay with only one method `remove` even now if `SecurityService` is injected into `CommunityUserCommandService`. But now no business service depends on `SecurityService`. All security related logic is in the controller layer.

If we decide to establish the dependency between business domain services and `CurrentUserService` it will be possible to unite these two methods into a single one.

Reply:
As commented before, we stick with security only on controller layer. You can leave the two methods as they are. It's OK. 👍